tests/unit-compatibility-test/fixtures/sonarqube-dce/change-admin-password-hook-values.yaml

```diff
@@ -588,8 +568,6 @@ spec:
               weight: 1
         nodeAffinity:
           
-      securityContext:
```

Comment:
Highlighting: in the previous fixture, a node like this was set to `secutiryContext: null`; in this case, it was removed entirely.